sql-database/failover-groups/add-single-db-to-failover-group-az-ps.ps1

```diff
@@ -0,0 +1,130 @@
+# Set variables for your server and database
+$subscriptionId = '<SubscriptionID>'
+$resourceGroupName = "myResourceGroup-$(Get-Random)"
+$location = "West US 2"
+$adminLogin = "azureuser"
+$password = "PASSWORD"+(New-Guid).Guid
+$serverName = "mysqlserver-$(Get-Random)"
+$databaseName = "mySampleDatabase"
+$drLocation = "East US 2"
+$drServerName = "mysqlsecondary-$(Get-Random)"
+$failoverGroupName = "failovergrouptutorial-$(Get-Random)"
+
+
+# The ip address range that you want to allow to access your server 
+# Leaving at 0.0.0.0 will prevent outside-of-azure connections
+$startIp = "0.0.0.0"
+$endIp = "0.0.0.0"
+
+# Show randomized variables
+Write-host "Resource group name is" $resourceGroupName 
+Write-host "Password is" $password  
+Write-host "Server name is" $serverName 
+Write-host "DR Server name is" $drServerName 
+Write-host "Failover group name is" $failoverGroupName
+
+
+# Connect to Azure
+Connect-AzAccount
+
+# Set subscription ID
+Set-AzContext -SubscriptionId $subscriptionId 
+
+# Create a resource group
+Write-host "Creating resource group..."
+$resourceGroup = New-AzResourceGroup -Name $resourceGroupName -Location $location -Tag @{Owner="SQLDB-Samples"}
+$resourceGroup
+
+
+# Create a server with a system wide unique server name
+Write-host "Creating primary logical server..."
+$server = New-AzSqlServer -ResourceGroupName $resourceGroupName `
+   -ServerName $serverName `
+   -Location $location `
+   -SqlAdministratorCredentials $(New-Object -TypeName System.Management.Automation.PSCredential `
+   -ArgumentList $adminLogin, $(ConvertTo-SecureString -String $password -AsPlainText -Force))
+$server
+
+# Create a server firewall rule that allows access from the specified IP range
+Write-host "Configuring firewall for primary logical server..."
+$serverFirewallRule = New-AzSqlServerFirewallRule -ResourceGroupName $resourceGroupName `
+   -ServerName $serverName `
+   -FirewallRuleName "AllowedIPs" -StartIpAddress $startIp -EndIpAddress $endIp
+$serverFirewallRule
+
+# Create General Purpose Gen5 database with 1 vCore
+Write-host "Creating a gen5 1 vCore database..."
+$database = New-AzSqlDatabase  -ResourceGroupName $resourceGroupName `
```

Comment:
Command caused the following error when tested on Az 2.4.0 / PowerShell 6.2.1 / UNIX (WSL)

```
Creating a gen5 1 vCore database...
New-AzSqlDatabase : The tier 'GeneralPurpose' does not support the sku 'SQLDB_GP_Gen5_1'.
At /mnt/c/Users/sttramer/src/azure-docs-powershell-samples/sql-database/failover-groups/add-single-db-to-failover-group-az-ps.ps1:49 char:13
+ $database = New-AzSqlDatabase  -ResourceGroupName $resourceGroupName  ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : CloseError: (:) [New-AzSqlDatabase], CloudException
+ FullyQualifiedErrorId : Microsoft.Azure.Commands.Sql.Database.Cmdlet.NewAzureSqlDatabase
```